src/style.css

```diff
@@ -2,0 +1,67 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.search-bar {
+  margin-top: 20px;
```

Comment:
It's recommended to be consistent with your margins. Here you've added a top margin, but in a real-world scenario, it's better to stick to just one direction (either top or bottom) to avoid potential margin collapse problems.